### DISCUSSION TITLE
The NewPipe icon look closer AIMP player

### DISCUSSION THREAD

**Discussion by TruongTanLuc201270** [TeamNewPipe/NewPipe#11145] [Questions]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

AIMP icon
![Aimp-icon](https://github.com/TeamNewPipe/NewPipe/assets/145727083/0baf8c7a-2508-45ac-9f3a-8a3bb52fa305)
AIMP icon [flipped]
![20240613_004217](https://github.com/TeamNewPipe/NewPipe/assets/145727083/5336a850-0e45-416a-babe-abbbd4169c51)
NewPipe icon but diffierent
![icon_OHy4y1W-fJCNhHHOBCM9V_cxZNJJgbcNkB-x7UDTY9Q=](https://github.com/TeamNewPipe/NewPipe/assets/145727083/34bff6a0-1054-444a-bdf6-40b53d7897c5)
AIMP player is mp3 player by Artem Izmaylov but NewPipe share icon closer.

### Additional information

Icon different
AIMP icon [flipped]
![20240613_004952](https://github.com/TeamNewPipe/NewPipe/assets/145727083/be835f54-c580-427f-804b-68f6b596c17f)
NewPipe icon
![20240613_005022](https://github.com/TeamNewPipe/NewPipe/assets/145727083/2847cc12-15ec-4169-b54d-eca246a3755c)

**Comment by AlexN1997** [comment]
What about the IGN player (from walkthroughs)
![Screenshot_20240613_185512_Firefox.png](https://github.com/TeamNewPipe/NewPipe/assets/76731207/149571ad-94ad-4583-bf6e-598c4fa348c4)
